modules/login/jsx/requestAccount.js

```diff
@@ -0,0 +1,243 @@
+          'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        body: send,
+      }
+    ).then((response) => response.json())
+      .then(
+        (data) => {
+          this.setState({request: true});
+        }).catch((error) => {
+      this.setState({request: true});
+    });
+  }
+
+  /**
+   * Used for including Google ReCaptcha.
+   */
+  loadGoogleCaptcha() {
+    /**
+     * Dynamically load a script if necessary.
+     * @param {string} url - script to load.
+     */
+    function loadScript(url) {
+      // Adding the script tag to the head as suggested before
+      let head = document.getElementsByTagName('head')[0];
+      const script = document.createElement('script');
+      script.type = 'text/javascript';
+      script.src = url;
+
+      // Start the loading...
+      head.appendChild(script);
+    }
+
+    // Include Google ReCaptcha.
+    loadScript('https://www.google.com/recaptcha/api.js');
+  }
+
+  /**
+   * @return {DOMRect}
+   */
+  render() {
+    const captcha = this.state.form.captcha ? (
+      <div className='form-group'>
+        <div className='g-recaptcha'
+             data-sitekey={this.state.form.captcha}/>
+        <span id='helpBlock' className='help-block'>
+          <b className='text-danger'>Please complete the reCaptcha!</b>
+        </span>
+      </div>
+    ) : null;
+    const request = !this.state.request ? (
+      <div>
+        <FormElement
+          name={'form1'}
+          action={''}
+          id={'form'}
+          fileUpload={'false'}
+          onSubmit={this.handleSubmit}
+        >
+          <StaticElement
+            text={'Please fill in the form below to request a LORIS account. ' +
+            'We will contact you once your account has been approved.'}
+            class={'col-sm-12'}
+            textClass={'text-center'}
+          />
+          <TextboxElement
+            name={'firstname'}
+            value={this.state.form.value.firstname}
+            onUserInput={this.setForm}
+            class={'col-sm-12'}
+            required={true}
+            type={'text'}
+            placeholder={'First name'}
+          />
+          <TextboxElement
+            name={'lastname'}
+            value={this.state.form.value.lastname}
+            onUserInput={this.setForm}
+            class={'col-sm-12'}
+            required={true}
+            type={'text'}
+            placeholder={'Last name'}
+          />
+          <TextboxElement
+            name={'email'}
+            value={this.state.form.value.email}
+            onUserInput={this.setForm}
+            class={'col-sm-12'}
+            required={true}
+            type={'text'}
+            placeholder={'Email address'}
+          />
+          <SelectElement
+            name={'site'}
+            options={this.props.data.site}
+            value={this.state.form.value.site}
+            onUserInput={this.setForm}
+            class={'col-sm-12'}
+            emptyOption={false}
```

Comment:
any way to add 'Choose your site:' as the "empty" option?

Reply:
@zaliqarosli I added some code to resolve this problem.